frontend/src/teacher/TeacherGallery.tsx

```diff
@@ -0,0 +1,30 @@
+import {Teacher} from "./Teacher";
+
+type TeacherGalleryProps = {
+    teachers : Teacher[]
+}
+
+export default function TeacherGallery (props: TeacherGalleryProps){
+
+
+    return(
+       <div>
+           <h2>aktuelle Lehrerliste</h2>
+           <table>
+               <tr>
+                   <th>Vorname</th>
+                   <th>Nachname</th>
+                   <th>Instrument</th>
+               </tr>
+
+               {props.teachers.map(teacher =>
+                   <tr>
```

Comment:
Immer wenn du mit .map eine List renderst muss das oberste element ein key prop mit einer eindeutigen id bekommen
```suggestion
                   <tr key={teacher.id}>
```